### DISCUSSION TITLE
MAX Hotend Temp ADJUSTMENT? Latest Build

### DISCUSSION THREAD

**Discussion by Riggiee** [Jyers/Marlin#1093] [Q&A] [answered]
Hi,

I have been using this build 'E3V2-UBL-BLTouch-15x15-HS-v4.2.2-v1.3.4.bin' very successfully. I only have one issue which is the max hotend temp is at 260C, i want to go above 300C.

Can anyone make this adjustment for me? i just cant do vscode properly

thanks

**Comment by Tuncay-Ayhan** [comment, marked as answer]
https://marlinfw.org/docs/basics/install_platformio_vscode.html


Download the Jyers firmware first.

https://github.com/Jyers/Marlin/archive/refs/heads/Ender_3_V2_JyersUI.zip

Unzip it.

And when I says this part:

**Drag your downloaded Marlin Firmware folder onto the Visual Studio Code application icon, or
Use the Open… command in the VSCode File menu, or
Open the PIO Home tab and click the “Open Project” button.**


Open the unzipped folder in VSCode
